net/frr/src/opnsense/service/templates/OPNsense/Quagga/+TARGETS

```diff
@@ -5,3 +5,4 @@ ripd.conf:/usr/local/etc/frr/ripd.conf
 frr:/etc/rc.conf.d/frr
 zebra.conf:/usr/local/etc/frr/zebra.conf
 vtysh.conf:/usr/local/etc/frr/vtysh.conf
+syslog-ng-frr-events.conf:/usr/local/syslog-ng.conf.d/frr-events.conf
```

Comment:
There's an /etc/ missing between

Reply:
oops, the one file I copied in manual for the test... thanks!